pages/common/z.md

```diff
@@ -26,3 +26,7 @@
 - Remove the current directory from `z`'s database:
 
 `z -x .`
+
+- Go to the directory that is a subdirectory of the current directory.
```

Comment:
@crs1138 it appears that `z -c` is the equivalent of `cd`.  If **and only if** this is true, please apply this suggestion:
```suggestion
- Go to the directory that is a subdirectory of the current directory (equivalent of `cd`):
```

Reply:
Thanks for your feedback. I've applied the changes you requested. Not sure how to proceed now. Can I resubmit this PR or better to make a new one?